src/app/window-handler.ts

```diff
@@ -1250,7 +1250,7 @@ export class WindowHandler {
       opts.alwaysOnTop = true;
     }
 
-    if (isWindowsOS && this.mainWindow) {
+    if (this.mainWindow) {
       opts.parent = this.mainWindow;
```

Comment:
On macOS this will attach the window to the main window amd ot won't be a separate window right?

Reply:
Here is a quick demo of its behaviour:

 ![2022-02-17 09 00 19](https://user-images.githubusercontent.com/51402489/154431355-8abf2b80-e4a6-40a1-9784-8e81fa368f54.gif)